shuttle/db/db_functions.py

```diff
@@ -162,3 +162,38 @@ def number_active_requests():
         return str(results[0]['COUNT(*)'])
     except:
         return "Error"
+      
+      
+def commit_driver_check_in(location,direction,driver_break):
+    now = datetime.datetime.now()
+    date = now.strftime('%d-%b-%Y')
+    full_date = now.strftime('%d-%b-%Y %I:%M %p')
+    username = flask_session['USERNAME']
+    single_quote = "\'"
+    if location != "":
+        if direction == 'departure':
+            sql = "INSERT INTO SHUTTLE_DRIVER_LOGS(LOG_DATE,USERNAME,LOCATION,DEPARTURE_TIME) VALUES (" + single_quote + \
+                  date + single_quote + "," + single_quote + username + single_quote + "," + single_quote + location + \
+                  single_quote + "," + "TO_DATE(" + single_quote + full_date + single_quote + ", \'dd-mon-yyyy hh:mi PM\'))"
+        elif direction == 'arrival':
+            sql = "INSERT INTO SHUTTLE_DRIVER_LOGS(LOG_DATE,USERNAME,LOCATION,ARRIVAL_TIME) VALUES (" + single_quote + \
```

Comment:
Also usually have spaces in the column names being inserted into. 
ie. (LOG_DATE, USERNAME, ...)

Also space between table name and those values
ie INSERT INTO table_name (column1, column2, column3, ...)

Looks like I missed this in your other sql queries so fix it there too please.